app/assets/stylesheets/components/editorToolbar.scss

```diff
@@ -0,0 +1,45 @@
+.editor-toolbar {
+  display: flex;
+  background: var(--base-0);
+  padding: var(--su-2) var(--content-padding-x);
+
+  &#overflow-menu {
+    position: asbolute;
+    background: revert;
+    right: var(--content-padding-x);
+    min-width: auto;
+  }
+
+  .formatter-btn {
+    width: 50px;
+
+    svg {
+      vertical-align: middle;
+    }
+
+    &__tooltip {
+      display: none;
+      position: absolute;
+      top: 100%;
+      left: 50%;
+      transform: translateX(-50%);
+      background-color: var(--syntax-background-color);
+      color: var(--base-0);
+      font-size: var(--fs-s);
+      padding: var(--su-1) var(--su-2);
+      border-radius: var(--radius);
+      width: max-content;
+    }
+
+    &__tooltip-shortcut {
+      color: var(--base-30);
+    }
+  }
+
+  .formatter-btn:hover,
+  .formatter-btn:focus-visible {
```

Comment:
We could probably update our crayons-tooltip component with that `:focus-visible` :P

Reply:
seems like everywhere we use crayons-tooltip it's on a non-interactive element so can't be focused 😢  I'll see if we need a wee a11y issue around that for later.